src/TestInternalGrains/ActivateDeactivateTestGrain.cs

```diff
@@ -23,13 +46,13 @@ internal class SimpleActivateDeactivateTestGrain : Grain, ISimpleActivateDeactiv
 
         public override async Task OnActivateAsync()
         {
-            this.logger = GetLogger();
+            logger = GetLogger();
```

Comment:
why all of these unnecessary changes? They make the diff harder. And personally, if anything, I would go the other way and add `this.` when it's not there. But that's a war I don't want to get into at all, as there can be 1 distinct opinion per contributor, but what I would say is let's not change code style for what it's already there even when the standard project style has not been set. Makes sense?